src/components/common/ProductBox/ProductBox.module.scss

```diff
@@ -75,4 +77,14 @@
     justify-content: space-between;
     align-items: center;
   }
+
+  &:hover{
+    .buttons{
+      opacity: 1;
+    }
+
```

Comment:
I see that you have in this PR commits from the older task. I assume, that you've started from older task branch. Never do that. Always start a new branch for a new task from `master`.

Reply:
Actually I did that. What should I do with that? Create completely new branch form master and start over?

Reply:
That is the "safest" option. Could you do that?

I'm closing this PR then.

Reply:
Use the name WDP20080202-2v2 for this new branch that you'll make.